packages/discovery-styles/scripts/update-styles.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
```

Comment:
Please add a comment block here explaining what this script does and how to invoke it (usage).